Layers/LayerDefinition.gd

```diff
@@ -0,0 +1,84 @@
+extends Resource
+class_name LayerDefinition
+
+signal z_index_changed(index)
+signal visibility_changed(visible)
+
+var geo_layer: RefCounted
+var crs_from:=3857
+var name: String
+
+class RenderInfo:
```

Comment:
maybe rename to `RenderInfo2D` to avoid confusion with `LayerComposition.RenderInfo` (which should be renamed to `RenderInfo3D`)